src/unitxt/operators.py

```diff
@@ -518,11 +515,12 @@ def process_value(self, value: Any) -> Any:
         words = re.split(r"(\s+)", value)
         new_value = ""
 
+        random_generator = new_random_generator(sub_seed=str(hash(value)))
```

Comment:
This means every value will have exactly the same augmentation. is that what we want?

Reply:
Another option for example, is that we will use the entire instance to determine the seed. Then it give some more freedom.

Reply:
So i can create a generator based on the entire instance in `Augmentor.process()`, and pass it here to `process_value`. sounds good?

Reply:
that's great

Reply:
There is a tradeoff - I think we should augment per value.  Indeed it means the same text will always augmented the same (given the global seed is the side). However, if we don't do this  adding a single element/removing to the list, will change the augmentation of all values.

Reply:
@elronbandel and i discussed, will keep as is (seed depends on value)